extensions/amp-analytics/0.1/vendors.js

```diff
@@ -79,5 +79,22 @@ export const ANALYTICS_CONFIG = {
     },
     'optout': '_gaUserPrefs.ioo'
   }
+  
+  'atinternet': {
+    'transport': {'beacon': false, 'xhrpost': false},
```

Comment:
Would it be best to be explicit here and include `'image': true` in this list?

cc @avimehta